solutions/fully-configurable/variables.tf

```diff
@@ -85,11 +85,10 @@ variable "service_endpoints" {
   }
 }
 
-
 variable "members" {
   type        = number
   description = "The number of members that are allocated. [Learn more](https://cloud.ibm.com/docs/messages-for-rabbitmq?topic=messages-for-rabbitmq-resources-scaling)."
-  default     = 3
+  default     = 2
```

Comment:
pipeline failed, it looks it has to be 3 after all. let's update it everywhere back to 3.